File: src/mobile/lib/app/utils/dto/account/account_dto.dart
class AccountDto {
  AccountDto({
    required this.id,
    required this.username,
    required this.email,
    this.phoneNumber,
    this.fotoUrl,
    required this.isAtivo,
  });
  late final String id;
  late final String username;
  late final String email;
  late final String? phoneNumber;
  late final String? fotoUrl;
  late final bool isAtivo;

  AccountDto.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    username = json['username'];
    email = json['email'];
    phoneNumber = json['phoneNumber'];
    fotoUrl = json['fotoUrl'];
    isAtivo = json['isAtivo'];
  }
}

File: src/services/Auth/Auth.API/Data/Dto/AccountDto.cs
namespace Auth.API.Data.Dto
{
  public record AccountDto
  {
    public string Id { get; set; } = null!;
    public string Username { get; set; } = null!;
    public string Email { get; set; } = null!;
    public string? PhoneNumber { get; set; }
    public string? FotoUrl { get; set; }
    public bool IsAtivo { get; set; }
  }
}